tf-modules/ingest/update-granules-cmr-metadata-file-links-task.tf

```diff
@@ -0,0 +1,33 @@
+resource "aws_lambda_function" "update_granules_cmr_metadata_file_links_task" {
+  function_name    = "${var.prefix}-UpdateGranulesCmrMetadataFileLinks"
+  filename         = "${path.module}/../../tasks/update-granules-cmr-metadata-file-links/dist/lambda.zip"
+  source_code_hash = filebase64sha256("${path.module}/../../tasks/update-granules-cmr-metadata-file-links/dist/lambda.zip")
+  handler          = "index.handler"
+  role             = var.lambda_processing_role_arn
+  runtime          = "nodejs12.x"
+  timeout          = 300
+  memory_size      = 1024
```

Comment:
We should probably consider tuning/eval of the timeout here, depending on what load/timing projections looks like for a reasonable amount of granules.   

 Definitely shouldn't block the PR, but move granules had the same setup and was S3 timing bound, system performance/etc will benefit from reducing the timeout for failure cases if we're well above what users would reasonably want.

Reply:
Good point! If I understand what you mean by `S3 timing bound`, this task similarly relies on S3 to retrieve an object. That would lead me to think that the timeout for this task is okay, but I am open to reducing the timeout for failure cases like you mentioned. @Jkovarik

Reply:
Sorry - what I was (poorly) getting at was generally we probably want timeouts to be trimmed as much as reasonably possible with padding so that config errors/bugs/etc that might cause a timeout don't eat invocation charges at scale -- given what the lambda is doing compared with what move granules was doing, I was wondering if in terms of "wildest reasonable expectations" if 5 minutes makes sense.

If you feel this is reasonable given task testing, no objection at all.  

Again, not a PR stopper at all, it's super minor given the scaling/budgets probably involved.